lib/src/feature/authentication/widgets/login_form.dart

```diff
@@ -27,6 +29,15 @@ class LoginForm extends StatelessWidget {
           textEditingController: controller.passwordTextEditingController,
           obscureText: true,
         ),
+        Spacer(),
+
+        /// Google Sign In button
+        GoogleSignInButton(
+          onPressed: () async {
+            await controller.signInWithGoogle();
+          },

```

Comment:
You can just do `GoogleSignInButton(onPressed: _controller.signInWithGoogle)`instead

Reply:
controller seems to be defined without the `_`underscore here, unlike in `landing_screen.dart`,
perhaps a little refactoring is needed in a future PR for consistency?
<img width="485" alt="Screen Shot 2022-08-21 at 1 09 57 PM" src="https://user-images.githubusercontent.com/31113245/185802744-394aebb8-ebca-44de-8f49-308101eb0898.png">
<img width="536" alt="Screen Shot 2022-08-21 at 1 10 37 PM" src="https://user-images.githubusercontent.com/31113245/185802774-0a526c81-1b64-4047-9be4-500cea4249fe.png">